assets/sass/_hero.scss

```diff
@@ -15,6 +15,7 @@
     }
   }
   .pledgebank & { background-color: #9c7bbd; }
+  .popit & { background-color: #2e353e; }
```

Comment:
When `rgba(255, 255, 255, 0.1)` is laid on top of this for the `.topbar`, you get a weird not-quite-invisible mySociety tab:

![screen shot 2018-07-11 at 14 23 27](https://user-images.githubusercontent.com/739624/42574403-3e8f9d00-8516-11e8-9195-e4a928932f35.png)

Grey also looks a bit sombre to me. Maybe we pick one of the colours out of the ferris wheel image?